### PR TITLE
[AJ-869] Bump listener hash for performance bugfix

### PR DESCRIPTION
**Pull request by rtitle**
https://broadworkbench.atlassian.net/browse/AJ-869

Listener PR: https://github.com/DataBiosphere/terra-azure-relay-listeners/pull/26

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,7 @@ EXPOSE 5050
 ENV GIT_HASH $GIT_HASH
 ENV HELM_DEBUG 1
 # WARNING: If you are changing any versions here, update it in the reference.conf
-ENV TERRA_APP_SETUP_VERSION 0.0.7
+ENV TERRA_APP_SETUP_VERSION 0.0.8
```
Comment (rtitle) [inline Dockerfile:26]: Setup chart bumped in https://github.com/DataBiosphere/terra-app/pull/32

File: http/src/main/resources/reference.conf [hunk 2]
```diff
@@ -964,7 +964,7 @@ terra-app-setup-chart {
   # cert files into the local chart.
   chart-name = "/leonardo/terra-app-setup"
   # If you change this here, be sure to update it in the dockerfile
```
Comment (aaronkanzer) [inline http/src/main/resources/reference.conf:966]: Is this docstring incorrect, outdated? Or does a value in a `Dockerfile` need to be bumped?

**Review by calypsomatic — APPROVED** [review]

**Comment by rtitle** [comment]
jenkins retest

**Comment by rtitle** [comment]
jenkins retest, some sbt SocketExceptions download dependencies..

**Comment by rtitle** [comment]
Only dataproc test failure due to permission propagation. Merging.

**Pull request merged**